test/parallel/test-util-inspect.js

```diff
@@ -2906,6 +2906,7 @@ assert.strictEqual(
     util.inspect(x, { maxStringLength: 4 }),
     "'aaaa'... 999996 more characters"
   );
+  assert(util.inspect(x, { maxStringLength: null }).endsWith("a'"));
```

Comment:
nit: use `assert.match` to improve the error message if the test fails?

```suggestion
  assert.match(util.inspect(x, { maxStringLength: null }), /a'$/);
```